egs/chime4/s5_1ch/local/compute_pesq.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# Copyright 2017 Johns Hopkins University (Author: Aswin Shanmugam Subramanian)
+# Apache 2.0
+
+set -e
+set -u
+set -o pipefail
+
+if [ $# != 4 ]; then
+   echo "Wrong #arguments ($#, expected 4)"
+   echo "Usage: local/compute_pesq.sh <enhancement-method> <enhancement-directory> <chime-rir-directory> <modeldir>"
```

Comment:
Please improve the documentation, see my comments on a similar script.

Reply:
Added documentation and example usage below the license.